testutils/src/main/scala/za/co/absa/enceladus/testutils/DataframeReader.scala

```diff
@@ -45,32 +46,37 @@ class DataframeReader(datasetPath: String, inputSchema: Option[StructType] )
     }
   }
 
-  private def getStandardReader()(implicit dfReaderOptions: DataframeReaderOptions, sparkSession: SparkSession): DataFrameReader = {
+  private def getStandardReader()(implicit dfReaderOptions: DataframeReaderOptions,
+                                  sparkSession: SparkSession): DataFrameReader = {
     sparkSession.read.format(dfReaderOptions.rawFormat)
   }
 
-  private def getParquetReader()(implicit dfReaderOptions: DataframeReaderOptions, sparkSession: SparkSession): DataFrameReader = {
+  private def getParquetReader()(implicit dfReaderOptions: DataframeReaderOptions,
+                                 sparkSession: SparkSession): DataFrameReader = {
     val dfReader = getStandardReader
 
-    if (inputSchema.isDefined) dfReader.schema(inputSchema.get)
-    else dfReader
+    if (inputSchema.isDefined) { dfReader.schema(inputSchema.get) }
```

Comment:
This is to please scalastyle, right?
I would do this with pattern matching:
```
inputSchema match {
  case Some(schema) => dfReader.schema(schema)
  case None         => dfReader
}
```
This way we get to keep the nice syntax of the line per condition without unnecessary parens and we get rid of `.isDefined` and `.get` calls.

Reply:
Done in all cases